app/src/main/java/com/github/se/eventradar/model/repository/user/FirebaseUserRepository.kt

```diff
@@ -182,6 +183,29 @@ class FirebaseUserRepository(db: FirebaseFirestore = Firebase.firestore) : IUser
     }
   }
 
+  override suspend fun addAttendingEvent(userId: String, attendingEventId: String): Resource<Unit> {
```

Comment:
nit: maybe a clearer name would be addAttendeeToEvent (hopefully the refactor feature does it all and it doesn't cause problems, but let me know what you think)

Reply:
yeah definetly needs an another name since it's "adding event to user attendance list" xD